.gitattributes

```diff
@@ -0,0 +1,3 @@
+/.* export-ignore
+/tests export-ignore
+/phpunit.xml.dist export-ignore
```

Comment:
is this needed, seems everything in root folder already is ignored through line 1